content/attacks/posts/2023-04-09-GDAC.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2023-04-09
+target-entities: GDAC
+entity-types:
+  - Custodian
+  - Exchange
+attack-types: Wallet Hack
+title: "GDAC Hacked for $13 Million"
+loss: 13000000
+---
+
+## Summary
+
+On April 9, 2023, South Korean cryptocurrency exchange GDAC was hacked, resulting in the theft of cryptocurrencies worth approximately $13 million. Hackers gained access to the exchange's hot wallets and stole various cryptocurrencies, including Bitcoin (BTC), Ethereum (ETH), and Wemix (WEMIX).
+
+## Attackers
+
+The attackers behind the GDAC hack remain unidentified.
+
+GDAC Hacker ETH wallets:
+- [0x244615D99684175d31369332039b2D84ce925EC5](https://etherscan.io/address/0x244615D99684175d31369332039b2D84ce925EC5)
+- [0x62B5eb2cb925Ce2898f9327B235b3228e7Cac1C2](https://etherscan.io/address/0x62B5eb2cb925Ce2898f9327B235b3228e7Cac1C2)
+- [0x87597bDB421482190e223aCa0A4DEAd75AB0a98D](https://etherscan.io/address/0x87597bDB421482190e223aCa0A4DEAd75AB0a98D)
+- [0x6225c189f583fc2e17f5be36a6a9db4d3978872d](https://etherscan.io/address/0x6225c189f583fc2e17f5be36a6a9db4d3978872d)
+- [0x57192cca8b8e4beb77f3466c6d0550e64cc53b0f](https://etherscan.io/address/0x57192cca8b8e4beb77f3466c6d0550e64cc53b0f)
```

Comment:
As usual, hackers utilizes a lot of addresses, please mention few of them or even single one, if there is only one marked on etherscan